hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1034,6 +1035,24 @@ public function upgrade_1025() {
   }
 
   /**
+   * Alters civicrm_hrjobcontract_hour table to make default values for fte_num
+   * and fte_denom equal to 0.
+   * 
+   * @return boolean
+   *   True on success
+   * @todo Rename upgrade!!  To upgrade_1026()  Include in upgrade bundle!
+   */
+  public function upgrade_1026() {
```

Comment:
Whenever you change the definition of a table with an upgrader method, you also need to update the sql script used during the extension installation. For hrjobcontract, this file is `sql/install.sql `.

The DAO has to be updated as well. Since there are no XML schemas for this extension, please update it manually.